en/docs/learn/provisioning-users-to-salesforce.md

```diff
@@ -0,0 +1,150 @@
+This tutorial will walk you through the steps on integrating WSO2 Identity Server with Salesforce for provisioning
+users by utilizing WSO2 Enterprise Integrator as the message coordinator. The use-case is that, whenever a user is
+created in the WSO2 Identity Server, the same user needs to be created as a ‘Lead’ in Salesforce.
+
+## Configuring Salesforce
+1.  Create a trial Developer account in Salesforce from  [here](https://login.salesforce.com/).
```

Comment:
Create a [trial Developer account](https://login.salesforce.com/) in Salesforce.